### ISSUE TITLE
Upgrade peer dependencies JSON Forms Package for vue2 to vue 2.7

### ISSUE THREAD

**Issue by marciado** [eclipsesource/jsonforms#1978]
### Is your feature request related to a problem? Please describe.

I want to upgrade Vue to Version 2.7.x. But I use the[ jsonforms/vue2](https://github.com/eclipsesource/jsonforms/tree/master/packages/vue2/vue2) package. This package has peer dependencies:
```
"peerDependencies": {
    "@jsonforms/core": "3.0.0-beta.5",
    "@vue/composition-api": "^1.4.2",
    "vue": "^2.6.14"
}
```

With this Peer dependencies I can not update to Vue 2.7.x

### Describe the solution you'd like

Can you please update peerDependencies to Vue 2.7.x and remove @vue/composition api. Because composition api is backported to Vue 2.7.

### Describe alternatives you've considered

-

### Framework

Vue 2

### RendererSet

Vanilla

**Comment by sdirix** [comment]
Hi @marciado,

The range "^2.6.14" includes "2.x" and therefore also "2.7.x". Is there a problem with the "@vue/composition-api" dependency or is this just about cleaning up the dependency?

**Comment by marcianos** [comment]
> Hi @marciado,
> 
> The range "^2.6.14" includes "2.x" and therefore also "2.7.x". Is there a problem with the "@vue/composition-api" dependency or is this just about cleaning up the dependency?

Thanks for your answer! When I try to update to vue 2.7.x I receive the following error:
![image](https://user-images.githubusercontent.com/14870642/179249865-ec1dde90-622a-4c5a-87e1-d045ac9da35e.png)

The composition api plugin should be removed because the composition api is back ported to vue 2.7 so you can directly import e.g. `import { defineComponent } from "vue";`.

**Comment by marciado** [comment]
> Hi @marciado,
> 
> The range "^2.6.14" includes "2.x" and therefore also "2.7.x". Is there a problem with the "@vue/composition-api" dependency or is this just about cleaning up the dependency?

Thanks for your answer! When I try to update to vue 2.7.x I receive the following error:
![image](https://user-images.githubusercontent.com/14870642/179249865-ec1dde90-622a-4c5a-87e1-d045ac9da35e.png)

The composition api plugin should be removed because the composition api is back ported to vue 2.7 so you can directly import e.g. `import { defineComponent } from "vue";`.

**Comment by marcianos** [comment]
> Ah I see, so it's not really our range but the composition api dependency explicitly forbids it. Makes sense to me.
> 
> Are you able to contribute the update to Vue 2.7?

I have created a PR #1981. Can you take a look at it @sdirix?

**Comment by sdirix** [comment]
Merged with #1981. Thanks for the contribution!

**Comment by marcianos** [comment]
> Merged with #1981. Thanks for the contribution!

Thanks! Do you know when it will be released?
